src/phoneField/PhoneField.story.mdx

```diff
@@ -0,0 +1,96 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks'
+
+import { PhoneField } from './index'
+
+<Meta title="Forms|PhoneField" />
+
+# **PhoneField**
+
+<Preview>
+  <Story name="Default">
+    <label id="phoneField" htmlFor="phoneField">
+      Accessibility label
+    </label>
+    <PhoneField id="phoneField" name="phoneField" onChange={() => {}} />
+  </Story>
+</Preview>
+
+### Custom
+
+<Preview>
+  <Story name="Custom">
+    <React.Fragment>
+      <label id="phoneFieldId" htmlFor="phoneFieldId">
+        Accessibility label
+      </label>
+      <PhoneField
+        id="phoneFieldId"
+        name="phoneField"
+        className="additionalClass"
+        aria-labelledby="labelId"
+        selectFieldLabel="Phone country code"
+        textFieldTitle="Phone number"
+        textFieldPlaceholder="Phone number"
+        defaultRegionValue="FR"
+        defaultPhoneValue="0778674534"
+        countriesWhiteList={['FR', 'ES']}
+        customCountriesName={{
+          FR: 'fr label',
+          ES: 'es label',
+        }}
+        isInline={false}
+      />
+    </React.Fragment>
+  </Story>
+</Preview>
+
+## Specifications
+
+This field consists of the input select field and regular input field.
+
+> Phone input
+
+Phone input part should only accept digits and trigger numeric keyboards on mobile devices.
+
+> Country phone code
+
+The country code should be displayed next to a country flag when selection is made.
+On the country list the full name plus code should be visible.
+
+## Behaviour
+
+On small devices the country selector and the phone input field should get separated into two separate fields.
+
+On large devices the inputs should be displayed as merged. Still the full name should be visible.
+
+Maximum width of the name should be in proportion of 1:2 to the number.
+If the name is longer than available field it should get truncated.
+
+## Usage
+
+```jsx
+import { PhoneField } from '@blablacar/ui-library/build/phoneField'
+
+<label><PhoneField name="phoneField" onChange={() => {}} /></label>
+<label htmlFor="ID">
```

Comment:
The `<label htmlFor="ID">` should not wrap the `PhoneField` component and should follow my ather remark above.